### PR TITLE
All products

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (458)
Screenshot (457)
-- This the product listing page where I have books of all the authors altogether.
-- This page has a header for navigation.
-- This page will have different cards for listed books.

--- I still have to add the FILTERS. These are just products.

### CODE REVIEW

File: site-pages/all_products/products.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.product-container{
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    width: 77%;
+}
+.product-cards-container{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```
Comment (stutisk) [inline site-pages/all_products/products.css:13]: By default ,flex Direction is row ,so this can be ignored

**Review by stutisk — APPROVED** [review]
Everything else looks good to me

File: site-pages/all_products/products.css (hunk 1 above)
Comment (Jia-A) [reply site-pages/all_products/products.css:13]: Oh yes, okay! Will remove it.

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! Apply the suggestion given by @stutisk and then you are good to merge

**Pull request merged**